concent_api/core/tests/utils.py

```diff
@@ -169,22 +189,23 @@ def _get_deserialized_report_computed_task(
     def _get_deserialized_task_to_compute(
         self,
         timestamp: Union[str, datetime.datetime, None] = None,
-        deadline = None,
+        deadline: Union[str, int, None] = None,
         task_id: str = '1',
         subtask_id: str = '2',
-        compute_task_def = None,
-        requestor_id: bytes = None,
-        requestor_public_key: bytes = None,
-        requestor_ethereum_public_key: bytes = None,
-        provider_id: bytes = None,
-        provider_public_key: bytes = None,
-        provider_ethereum_public_key: bytes = None,
-        price = 0,
+        compute_task_def: Union[ComputeTaskDef, None] = None,
```

Comment:
If you want to get [something, None] better way would be `Optional[something]` it means that you expect something or None

Reply:
will be done :)